data/data-pipeline/data_pipeline/etl/sources/hud_housing/etl.py

```diff
@@ -13,7 +13,8 @@ def __init__(self):
         self.HOUSING_ZIP_FILE_DIR = self.get_tmp_path() / "hud_housing"
 
         # We measure households earning less than 80% of HUD Area Median Family Income by county
-        # and paying greater than 30% of their income to housing costs.
+        # and paying greater than 30% of their income to housing costs, with the exception of renters.
```

Comment:
Shouldn't this be `and paying greater than 50% of their income on housing costs`?

Reply:
yes!